### DISCUSSION TITLE
Sort number columns that are in text format

### DISCUSSION THREAD

**Discussion by tqdo** [TanStack/table#2840] [Q&A]
I am trying to make sure columns that have data like this: '1,234,567 M' get sorted correctly when users click the header. 
I found this https://github.com/tannerlinsley/react-table/issues/622 and but it requires me to convert my data to numerical, ie: 1234567. Is there anyway I can keep my data shown in the table as '1,234,567 M' but use 1234567 when sorting. Thanks

**Comment by saevarb** [comment]
The presentation of your data is entirely up to you. Simply create a component which renders that cell as you wish.
